apps/coscrad-frontend/src/utils/generic-components/presenters/tables/generic-index-table-presenter/index-table.tsx

```diff
@@ -144,83 +152,107 @@ export const IndexTable = <T extends IBaseViewModel>({
-                            onClick={() =>
-                                setCurrentPageIndex(
-                                    cyclicIncrement(currentPageIndex, lastPageIndex + 1)
-                                )
-                            }
-                        >
-                            Next
-                        </ArrowForwardIosIcon>
-                    </Paper>
-                </Typography>
-            </div>
+                            </TableBody>
+                        </Table>
+                    </TableContainer>
+                    <Box
+                        component="div"
+                        sx={{
+                            display: 'grid',
+                            flexDirection: 'row',
+                            alignItems: 'center',
+                            justifyItems: 'flex-end',
+                        }}
+                    >
+                        <Grid container justifyContent="flex-end" spacing={3}>
+                            <Grid item sx={{ display: 'flex', alignItems: 'center' }}>
+                                <Typography component="span" sx={{ mr: 2 }}>
+                                    Rows per page:
+                                </Typography>
+                                <FormControl variant="standard" sx={{ m: 1 }}>
+                                    <Select
+                                        name="pageSize"
+                                        value={pageSize}
+                                        onChange={(changeEvent) => {
+                                            const {
+                                                target: { value },
+                                            } = changeEvent;
+
+                                            const newPageSize =
+                                                typeof value === 'string'
+                                                    ? Number.parseInt(value)
+                                                    : value;
+
+                                            setPageSize(newPageSize);
+                                        }}
+                                    >
+                                        {pageSizeOptions.map((pageSize) => (
+                                            <MenuItem key={pageSize} value={pageSize}>
+                                                {pageSize}
+                                            </MenuItem>
+                                        ))}
+                                    </Select>
+                                </FormControl>
+                            </Grid>
+                            <Grid item sx={{ display: 'flex', alignItems: 'center' }}>
+                                Page: {currentPageIndex + 1}/{lastPageIndex + 1}
+                            </Grid>
+                            <Grid item sx={{ display: 'flex', alignItems: 'center' }}>
+                                <IconButton>
```

Comment:
I'm trying to figure out why the `ArrowBackIosIcon` isn't quite centred in the `IconButton `whearas the `ArrowForwardIosIcon `is centred

Reply:
Not sure I see what you're referring to, like centered horizontally or vertically?